website/blog/2022-07-12-change-data-capture.md

```diff
@@ -110,7 +110,7 @@ Her goal is to capture **all** versions of the `fct_income` model for January. S
 | January 2022 | electronics | 152 | 02/10/22 08:00:00 |
 | January 2022 | books | 202 | 02/10/22 08:00:00 |
 
-In order to achieve this **long table of history**, she decides to start [snapshotting](https://docs.getdbt.com/docs/building-a-dbt-project/snapshots) `fct_income`.
+In order to achieve this **long table of history**, she decides to do something that ⚠️ goes against dbt Lab's best practices ⚠️ - [snapshotting](https://docs.getdbt.com/docs/building-a-dbt-project/snapshots) her final model, `fct_income`.
```

Comment:
I was thinking we should use a Docusaurus [admonition](https://docusaurus.io/docs/next/markdown-features/admonitions), to get some visual callout for anyone who is just skimming. Do you think it's overkill? 

```suggestion
In order to achieve this **long table of history**, she decides to start [snapshotting](https://docs.getdbt.com/docs/building-a-dbt-project/snapshots) her final model, `fct_income`.

:::caution Don't be like Joanne

I'm including the code samples for completeness, but remember: the method described in this scenario contradicts dbt best practices. Either of the [solutions](#solution-1-downstream-incremental-model) [detailed](#solution-2-upstream-snapshots) later in the article is a better approach.

:::
```